integreat_cms/core/signals/hix_signals.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from ...cms.models import PageTranslation
+from ...cms.views.utils.hix import lookup_hix_score
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(post_save, sender=PageTranslation)
+def page_translation_save_handler(instance, **kwargs):
+    r"""
+    Calculates a hix store for a page translation after saving
+
+    :param instance: The page translation that gets saved
+    :type instance: ~integreat_cms.cms.models.pages.page_translation.PageTranslation
+
+    :param \**kwargs: The supplied keyword arguments
+    :type \**kwargs: dict
+    """
+    if instance.hix_score or not hix_enabled(instance):
+        return
+
+    if score := lookup_hix_score(instance.content):
+        logger.debug(
+            "Storing hix score %s for new page translation %s", score, instance.title
```

Comment:
```suggestion
            "Storing hix score %s for new page translation %r", score, instance
```
I'd prefer the repr for logging instead only the title.